README.md

```diff
@@ -1,7 +1,111 @@
-### ReactiveFeedback
+# ReactiveFeedback
 
 Unideraction reactive architecture. This is [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) implemetation of the [RxFeedback](https://github.com/kzaher/RxFeedback)
```

Comment:
unidirectional

Reply:
This is `a` [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift) implemetation of ~the~ [RxFeedback](https://github.com/kzaher/RxFeedback)